spec/controllers/api/workspaces_controller_spec.rb

```diff
@@ -0,0 +1,87 @@
+require 'rails_helper'
+
+RSpec.describe Api::WorkspacesController, type: :controller do
+  render_views
+
+  let!(:user) { create(:user) }
+  let!(:user1) { create(:user) }
+  let!(:user2) { create(:user) }
+  let!(:user3) { create(:user) }
+  let!(:workspace) { create(:workspace, user: user) }
+  let!(:workspace1) { create(:workspace, user: user1) }
+
+  before do
+    sign_in user
+  end
+
+  context 'GET /workspaces' do
+    it 'should show all user workspaces' do
+      get :index, as: :json
+      expect(response).to have_http_status(200)
+    end
+  end
+
+  context 'GET /worspaces/:id' do
+    it 'should show page with workspace' do
+      get :show, as: :json, params: { id: workspace.id }
+      parsed_response = JSON.parse(response.body)
+      expect(response.body).to include(workspace.name)
```

Comment:
I think here you can specify something like 
expect(response.body.something.name).to eql(workspace.name)